context/app/static/js/components/entity-search/searchkit-modifications/withSearchkitRouting.js

```diff
@@ -0,0 +1,166 @@
+// Copied from https://github.com/searchkit/searchkit/blob/6d11b204520009a705fe207535bd4f18d083d361/packages/searchkit-client/src/withSearchkitRouting.tsx
+// Modified to handle custom route state
+
+/* eslint-disable radix */
+/* eslint-disable react-hooks/rules-of-hooks */
+import React, { useEffect } from 'react';
+import isEqual from 'fast-deep-equal';
+import qs from 'qs';
```

Comment:
huh: we weren't already getting these as transitive dependencies from searchkit?

Reply:
Yes, but eslint complains about not having them specifically listed as dependencies. 

https://nodejs.org/es/blog/npm/peer-dependencies/